src/Fusion.Summary.Functions/Functions/DepartmentResourceOwnerSync.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Fusion.Resources.Functions.Common.ApiClients;
+using Fusion.Resources.Functions.Common.ApiClients.ApiModels;
+using Microsoft.Azure.WebJobs;
+
+namespace Fusion.Summary.Functions.Functions;
+
+public class DepartmentResourceOwnerSync
+{
+    private readonly ILineOrgApiClient lineOrgApiClient;
+    private readonly ISummaryApiClient summaryApiClient;
+
+    public DepartmentResourceOwnerSync(ILineOrgApiClient lineOrgApiClient, ISummaryApiClient summaryApiClient)
+    {
+        this.lineOrgApiClient = lineOrgApiClient;
+        this.summaryApiClient = summaryApiClient;
+    }
+
+    [FunctionName("department-resource-owner-sync")]
+    public async Task RunAsync(
+        [TimerTrigger("0 05 00 * * *", RunOnStartup = false)]
+        TimerInfo timerInfo, CancellationToken cancellationToken
+    )
+    {
+        var departments = await lineOrgApiClient.GetOrgUnitDepartmentsAsync();
+
+        var selectedDepartments = departments
+            .Where(d => d.FullDepartment != null).DistinctBy(d => d.SapId).ToList();
+
+        if (!selectedDepartments.Any())
+            throw new Exception("No departments found.");
+
+        var resourceOwners = new List<LineOrgPerson>();
+        foreach (var orgUnitsChunk in selectedDepartments.Chunk(10))
+        {
+            var chunkedResourceOwners =
```

Comment:
This is something that @larfeq is changing in another pr, but for now this should be ok. Maybe add a todo or something where we change the way we get resourceowners